packages/main/src/Input.js

```diff
@@ -705,6 +706,10 @@ class Input extends UI5Element {
 		this.fireEvent(this.EVENT_CHANGE);
 	}
 
+	_handlePaste(event) {
```

Comment:
The eslint will complain and the build will continue failing, because the "event" param as it is not used, you would have to remove it.

Reply:
Done